recipe/meta.yaml

```diff
@@ -1,29 +1,29 @@
-{% set version = "2.12.1" %}
+{% set version = "2.13.3" %}
 
 package:
   name: freetype
   version: {{ version }}
 
 source:
   url: https://download.savannah.gnu.org/releases/freetype/freetype-{{ version }}.tar.gz
-  sha256: efe71fd4b8246f1b0b1b9bfca13cfff1c9ad85930340c27df469733bbb620938
-  patches:
-    - CVE-2022-37434.patch
+  sha256: 5c3a8e78f7b24c20b25b54ee575d6daa40007a5f4eea2845861c3409b3021747
```

Comment:
Saw a couple checksum failures in CI. Think the mirror may be having some issues. As it does appear to be mostly working, have left it alone. However if this persists, we may want to switch to a different mirror